content/changelog-preview/index.html.haml

```diff
@@ -0,0 +1,19 @@
+---
+layout: changelog
+title: Changelog preview
+show_ratings: true
+has_rss: true
+---
+%p
+  %b
+    This is a preview changelog for upcoming releases. For publicly available releases see
+    %a{:href => '/changelog/'}
+      the weekly release changelog.

```

Comment:
Discussed minor phrasing change as our suggestion to link to weekly and to LTS changelogs so that those who see the site will know that they other changelogs are for releases.
```suggestion
    This is a preview changelog for the upcoming weekly release.
    For published weekly releases, see the
    %a{:href => '/changelog/'}
      weekly release changelog
    .
    For long term support releases, see the
    %a{:href => '/changelog-stable/'}
      long term support changelog
    .
```